src/main/java/ee/tuleva/onboarding/epis/EpisService.java

```diff
@@ -210,7 +210,8 @@ private HttpHeaders createJsonHeaders() {
   }
 
   private String userJwtToken() {
-    final var authentication = SecurityContextHolder.getContext().getAuthentication();
+    final var authentication =
+        SecurityContextHolder.getContext().getAuthentication(); // Use Authentication Holder Ath
```

Comment:
What's Ath?

Reply:
I think Jordan wrote this, seems to be a misspelling of Auth. Will fix